modules/checklist/assets/js/components/header.js

```diff
@@ -0,0 +1,43 @@
+import Typography from '@elementor/ui/Typography';
+import CloseButton from '@elementor/ui/CloseButton';
+import AppBar from '@elementor/ui/AppBar';
+import Divider from '@elementor/ui/Divider';
+import Toolbar from '@elementor/ui/Toolbar';
+import { __ } from '@wordpress/i18n';
+
+const Header = ( props ) => {
+	const { setIsOpen } = props;
+
+	return (
+		<>
+			<AppBar
+				elevation={ 0 }
+				position="sticky"
+				sx={ {
+					backgroundColor: 'background.default',
```

Comment:
Maybe instead of setting a background color, set color="transparent"? to get the background  behind it?:
https://elementor.github.io/elementor-ui/?path=/story/surfaces-appbar--basic&args=color:transparent

Reply:
done